### DISCUSSION TITLE
CMD + < for toggling between windows

### DISCUSSION THREAD

**Discussion by globalexport** [zed-industries/zed#17345] [Feature Requests]
Hi! 

I am using a German keyboard on MacOS where I have a global shortcut CMD + < to toggle between windows as both keys are next to each other. 

This does not work in zed when focusing the editor panel. It works once after manually focusing the project panel, but after the switch the focus is on the editor again. 

I failed to send the shortcut to the ProjectPanel in the keymap.json file like so: 

```
[
  {
    "context": "Editor",
    "bindings": {
      "cmd-<": ["ProjectPanel::SendKeystrokes", "cmd-<"]
    }
  }
]
```
Do you have an idea why it is not working?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
